### PR TITLE
i18n: dt savage cn translations fix

### PR DESCRIPTION
**Pull request by lsl1225**

### CODE REVIEW

File: ui/raidboss/data/07-dt/raid/r1s.ts [hunk 1]
```diff
@@ -556,13 +556,13 @@ const triggerSet: TriggerSet<Data> = {
...
           ja: '中へ + ヒラ頭割り => 外へ',
-          cn: '场内 + 治疗组分摊 => 场外',
+          cn: '场内 + 治疗分摊组 => 场外',
         },
         out: {
           en: 'Out + Healer Stacks => In',
           de: 'Raus + Auf Heiler sammeln => Rein',
           ja: '外へ + ヒラ頭割り => 中へ',
-          cn: '场外 + 治疗组分摊 => 场内',
+          cn: '场外 + 治疗分摊组 => 场内',
```
Comment (Echoring) [inline ui/raidboss/data/07-dt/raid/r1s.ts:565]: `治疗分摊组` is not fluent, I believe the previous one is better, short for `按治疗分组分摊`.

File: ui/raidboss/data/07-dt/raid/r4s.ts [hunk 2]
```diff
@@ -1004,19 +1004,19 @@ const triggerSet: TriggerSet<Data> = {
...
           ja: '自分から遠い人に扇範囲',
-          cn: '远雷点名',
+          cn: '远水波点名',
         },
         proximateCurrent: {
           en: 'Near Cone on You',
           de: 'Nah-Kegel auf DIR',
           ja: '自分から近い人に扇範囲',
-          cn: '近雷点名',
+          cn: '近水波点名',
```
Comment (Echoring) [inline ui/raidboss/data/07-dt/raid/r4s.ts:1013]: `水波` mainly give an impression of `万变水波` like mech, and this case is actually `雷`。

File: ui/raidboss/data/07-dt/raid/r4s.ts [hunk 3]
```diff
@@ -1028,7 +1028,7 @@ const triggerSet: TriggerSet<Data> = {
           en: 'Get Hit by Cone',
           de: 'Werde vom Kegel getroffen',
           ja: '扇範囲に当たって',
-          cn: '去吃雷',
+          cn: '吃水波',
```
Comment (Echoring) [inline ui/raidboss/data/07-dt/raid/r4s.ts:1031]: same as above.

File: ui/raidboss/data/07-dt/raid/r4s.ts [hunk 4]
```diff
@@ -1178,13 +1178,13 @@ const triggerSet: TriggerSet<Data> = {
...
           ja: 'デバフを渡して',
-          cn: '传递 Debuff',
+          cn: '传火',
         },
         getDebuff: {
           en: 'Get Debuff',
           de: 'Debuff nehmen',
           ja: 'デバフを取って',
-          cn: '获取 Debuff',
+          cn: '拿火',
```
Comment (Echoring) [inline ui/raidboss/data/07-dt/raid/r4s.ts:1187]:
```suggestion
          cn: '接火',
```

File: ui/raidboss/data/07-dt/raid/r4s.ts [hunk 5]
```diff
@@ -1249,7 +1249,7 @@ const triggerSet: TriggerSet<Data> = {
             en: 'LB3!',
             de: 'LB3!',
             ja: 'タンク LB3!',
-            cn: '坦克 LB!',
+            cn: '坦克LB!',
```
Comment (Echoring) [inline ui/raidboss/data/07-dt/raid/r4s.ts:1252]: There should be a blank between Chinese charactor and English charactor.

File: ui/raidboss/data/07-dt/raid/r4s.ts [hunk 6]
```diff
@@ -1550,7 +1550,7 @@ const triggerSet: TriggerSet<Data> = {
           en: '${side}: Start at ${first}',
           de: '${side}: Starte ${first}',
           ja: '${side}: まずは ${first} から',
-          cn: '${side}: 从 ${first} 开始',
+          cn: '${side}: 从${first}开始',
```
Comment (Echoring) [inline ui/raidboss/data/07-dt/raid/r4s.ts:1553]: Same as above.

File: ui/raidboss/data/07-dt/raid/r4s.ts [hunk 7]
```diff
@@ -1596,7 +1596,7 @@ const triggerSet: TriggerSet<Data> = {
           en: '${side} Side: ${order}',
           de: '${side} Seite: ${order}',
           ja: '${side} : ${order}',
-          cn: '${side} 侧: ${order}',
+          cn: '${side}侧: ${order}',
```
Comment (Echoring) [inline ui/raidboss/data/07-dt/raid/r4s.ts:1599]: Same as above.

File: ui/raidboss/data/07-dt/raid/r1s.ts (hunk 1 above)
Comment (Echoring) [reply ui/raidboss/data/07-dt/raid/r1s.ts:565]: Fine, standard output use this expression.

File: ui/raidboss/data/07-dt/raid/r4s.ts (hunk 2 above)
Comment (lsl1225) [reply ui/raidboss/data/07-dt/raid/r4s.ts:1013]: I want to let people know this is `万变水波` liked mech so I made these changes previously. Maybe callout original mech is better. These changes will be reverted to current callouts!

Comment (Echoring) [reply ui/raidboss/data/07-dt/raid/r4s.ts]: In fact, I believe `万变水波` is more likely to describe `诱导扇形后回避二段复读` mech (sometimes also used as `八方分散`), and this mech is a simple far/close bait, not belongs to `万变水波` type?.

**Review by Echoring — APPROVED** [review]

File: ui/raidboss/data/07-dt/raid/r4s.ts (hunk 2 above)
Comment (lsl1225) [reply ui/raidboss/data/07-dt/raid/r4s.ts]: Yep, it has difference between these mechanics after thinking twice, so leave it unchanged by now! Thanks!

**Pull request merged**